app/src/main/res/layout/fragment_second.xml

```diff
@@ -25,78 +24,158 @@
         android:id="@+id/fullname"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="4dp"
-        android:layout_marginLeft="4dp"
-        android:layout_marginTop="352dp"
-        android:background="#F5F5F8"
+        android:layout_marginStart="20dp"
+        android:layout_marginLeft="20dp"
+        android:layout_marginTop="310dp"
```

Comment:
we do not need that amount of margin, try to find a constraint. maybe an item bellow the other could fix your issue.